src/claimtrie.cpp

```diff
@@ -1083,6 +1137,15 @@ bool CClaimTrie::InsertFromDisk(const std::string& name, CClaimTrieNode* node)
         current = itchild->second;
     }
     current->children[name[name.size()-1]] = node;
+
+    // NOTE: add all claims if we want non-winning claims in the claim
```

Comment:
all claims should go into the index, even nonwinning ones. When I run `getclaimbyid`, it doesn't care about whether that claim is winning.